weatherreport.cpp

```diff
@@ -19,66 +21,90 @@ class IWeatherSensor {
 /// without needing the actual Sensor during development
 
 class SensorStub : public IWeatherSensor {
-    int Humidity() const override {
-        return 72;
-    }
+  int Humidity() const override {
+    return 72;
+  }
 
-    int Precipitation() const override {
-        return 70;
-    }
+  int Precipitation() const override {
+    return 70;
+  }
 
-    double TemperatureInC() const override {
-        return 26;
-    }
+  double TemperatureInC() const override {
+    return 26;
+  }
 
-    int WindSpeedKMPH() const override {
-        return 52;
-    }
+  int WindSpeedKMPH() const override {
+    return 52;
+  }
 };
 
+
+class mySensorStub :public IWeatherSensor{
```

Comment:
Another option is to have a single Stub class with the values passed in the constructor, which the functions will return.
That way you don't need to make new classes for every situation